src/anshuKJun20/ScannerEX.java

```diff
@@ -0,0 +1,265 @@
+package anshuKJun20;
+
+import java.util.Scanner;
+
+import javax.annotation.processing.SupportedSourceVersion;
+
+public class ScannerEX {
+	 static int countICICIcredit,countICICIdebit,counthdfccredit,counthdfcdebit,countsbicredit,countsbidebit,countbobcredit,countbobdebit ,totalcreditoperation,totaldebitoperation;
+	  static int hdfcbankbalance = 30000,icicibankbalance=4000,sbibankbalance=10000,bobbankbalance=5000;
```

Comment:
avoid giving hardcode value from next time. you should write a setData method to set initial balance.